packages/block-editor/src/components/block-list/style.scss

```diff
@@ -85,6 +85,21 @@
 			opacity: 1;
 		}
 	}
+
+	// Active template part highlighting.
```

Comment:
We could update this to refer to spotlight entities and mention template parts as one specific example.